assets/css/layout/_footer.scss

```diff
@@ -0,0 +1,242 @@
+// Footer styles.
+
+footer {
+  background: $navy-light;
+  color: $white;
+  margin-top: rem(30);
+  padding: rem(16) rem(32);
+
+  .netlify-notice {
+    padding-top: rem(32);
+    text-align: center;
+
+    a {
+      color: $white;
+      font-weight: $font-weight-normal;
+
+      &:hover,
+      &:focus {
+        color: $navy-light;
+      }
+    }
+  }
+
+  .footer-content {
+    align-items: baseline;
+    display: flex;
+    flex-direction: column;
+    font-size: rem(16);
+    line-height: rem(29);
+    margin: auto;
+    max-width: $max-width-margins;
+
+    h2 {
+      color: $white;
+      font-size: rem(16);
+      font-weight: bold;
+    }
+
+    .funders {
+      p {
+        margin: 0;
+      }
+
+      a {
+        color: $white;
+      }
+
+      svg {
+        padding: rem(13) rem(13) rem(13) 0;
+        width: rem(144);
+      }
+    }
+
+    .contact-info {
+      width: 100%;
+
+      address {
+        font-style: normal;
+      }
+
+      a {
+        color: $white;
+        font-weight: $font-weight-normal;
+
+        svg {
+          padding: rem(32) rem(16) rem(16) 0;
+          width: rem(240);
+        }
+      }
+
+      .logo-ocadu-wrapper {
+        text-align: center;
+      }
+
+      .logo-ocadu {
+        a svg {
+          padding: rem(16);
+          width: rem(128);
+        }
+      }
+    }
+
+    .footer-nav {
+      display: none;
+    }
+
+    .footer-nav,
+    .contact-info,
+    .funders {
+      a:hover,
+      a:focus {
+        color: $navy-light;
+      }
+    }
+
+    .social-media {
+      align-self: center;
+      padding-bottom: rem(32);
+
+      .social-media-icons {
+        display: grid;
+        grid-row-gap: rem(16);
+        grid-template-columns: repeat(2, auto);
+        margin: auto;
+        width: rem(144);
+
+        div {
+          justify-self: center;
+        }
+
+        a {
+          svg {
+            height: rem(60);
+            stroke-width: 3px;
+            width: rem(60);
+          }
+
+          &:hover,
+          &:focus {
+            background-color: inherit;
+
+            svg {
+              fill: $navy-light;
+              stroke: $white;
+              stroke-linecap: round;
+
+              path {
+                fill: $navy-light;
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  footer {
+    margin-top: rem(80);
+
+    .footer-content {
+      display: grid;
+      grid-row-gap: rem(16);
+      grid-template-columns: 3.8fr 1.2fr;
+      padding: rem(32) rem(48);
+
+      .footer-nav {
+        display: block;
+
+        a {
+          color: $green-light;
+          font-weight: $font-weight-semibold;
+        }
```

Comment:
There is no current page indicator on the footer link styles. The header navigation adds a `border-bottom` to links with the `.nuxt-link-active` class.

I also noticed that the Home link in the footer has a `.nuxt-link-active` class on all pages. This seems like a bug.

Lastly, the `.nuxt-link-active` class doesn't provide any info to assistive technology users. The correct way of indicating that a link represents the current page is using `aria-current="page"`. I'm wondering if the `<NuxtLink>` component supports this?

Reply:
> There is no current page indicator on the footer link styles. The header navigation adds a `border-bottom` to links with the `.nuxt-link-active` class.

It's decided at yesterday's meeting that using `aria-current="page"` for the active link on the footer is enough and doesn't need to add a visual indicator.